src/integration/pages/storyPage/crossPlatformIncludeTests.js

```diff
@@ -0,0 +1,19 @@
+export default () => {
+  describe('Includes', () => {
+    describe('IDT2', () => {
+      it('I can see a fallback image', () => {
```

Comment:
I think I'd prefer it if this test was duplicated on canonical as it is only temporary, mainly so the `it` can explain why we are showing a fallback image on canonical: `it('I can see a fallback image because the include uses disallowed scripts')`.

Reply:
Cool - happy to reintroduce this due to the temporary nature of the duplicated test. I've added a comment for further clarification.